docs/java/guides/tutorial-overview-sdk-java.md

```diff
@@ -0,0 +1,57 @@
+---
+id: tutorial-overview-sdk-java
+title: Tutorials for the SAP Cloud SDK for Java
+hide_title: false
+hide_table_of_contents: false
+sidebar_label: Tutorials Overview
+description: This document will give you an overview of the different tutorials for the SAP Cloud SDK for Java.
+---
+
+This document gives you an overview of which tutorials, tutorial groups and tutorial missions are available for the SAP Cloud SDK for Java.
+
+## Tutorial Overview
+
+Tutorials can be grouped. A group is a series of individual tutorials. A mission consists of groups and/or individual tutorials. In this way you can carry out a more extensive and meaningful task.
+
+### Missions
+
+Missions are the first point of contact if you want to familiarize yourself with the SAP Cloud SDK for Java and want to achieve certain goals.
+
+[Create a Cloud Foundry App Using SAP Cloud SDK](https://developers.sap.com/mission.cloudsdk-cf-app.html)
+
+### Groups
+
+Groups are a collection of topic-based tutorials.
+
+[Add Security and More to Your SAP Cloud SDK App](https://developers.sap.com/group.cloudsdk-more-features.html)
+
+[Create a Neo App Using SAP Cloud SDK](https://developers.sap.com/group.s4sdk-neo.html)
+
+[Create a Simple Cloud Foundry App Using SAP Cloud SDK](https://developers.sap.com/group.s4sdk-cloud-foundry.html)
+
+
+### Tutorials
+
+[Connect to OData Service on Cloud Foundry Using SAP Cloud SDK](https://developers.sap.com/tutorials/s4sdk-odata-service-cloud-foundry.html)
+
+[Connect to OData service on Neo using SAP Cloud SDK](https://developers.sap.com/tutorials/s4sdk-odata-service-neo.html)
+
+[Consume Messages for Business Events from SAP S/4HANA Cloud](https://developers.sap.com/tutorials/s4sdk-consuming-messages.html)
+
+[Create a Sample Application on Cloud Foundry Using SAP Cloud SDK](https://developers.sap.com/tutorials/s4sdk-cloud-foundry-sample-application.html)
+
+[Create a Sample Application on SCP Neo Using SAP Cloud SDK](https://developers.sap.com/tutorials/s4sdk-scp-neo-sample-application.html)
+
+[Create Cloud Foundry App with Blockchain Features of SAP Cloud SDK](https://developers.sap.com/tutorials/blockchain-sdk-sample.html)
+
+[Extensibility, Type-Safe Expand, and Dependency Injection with the Virtual Data Model for OData](https://developers.sap.com/tutorials/cloudsdk-extensibility-type-safe-expand.html)
+
+[Introduce Caching to Your Application](https://developers.sap.com/tutorials/s4sdk-caching.html)
+
+[Introduce Resilience to your Application](https://developers.sap.com/tutorials/s4sdk-resilience.html)
+
+[Migrate an Existing Application to Version 3.0 of the Cloud SDK](https://developers.sap.com/tutorials/s4sdk-migration-v3.html)
+
+[Secure Your Application on SAP Cloud Platform Cloud Foundry](https://developers.sap.com/tutorials/s4sdk-secure-cloudfoundry.html)
+
+[Set Up Your Local Infrastructure to Develop with SAP Cloud SDK](https://developers.sap.com/tutorials/s4sdk-setup.html)
```

Comment:
I would prefer it as a list with subheadings from our tutorials portal as a description. It can be done later, but prefer to have it already.
![image](https://user-images.githubusercontent.com/10460752/83279445-669ea380-a1d5-11ea-92d7-b3c1ecc5bf16.png)